perception/occupancy_map_monitor/include/moveit/occupancy_map_monitor/occupancy_map_updater.h

```diff
@@ -107,6 +107,8 @@ class OccupancyMapUpdater
 
   static void readXmlParam(XmlRpc::XmlRpcValue &params, const std::string &param_name, double *value);
   static void readXmlParam(XmlRpc::XmlRpcValue &params, const std::string &param_name, unsigned int *value);
+  static void readXmlParam(XmlRpc::XmlRpcValue &params, const std::string &param_name, double *value, double default_value);
```

Comment:
these actually aren't needed -- if the param does not exist, the value will not be updated (and thus will remain at whatever previous value it had)